src/components/QuickLinks/QuickLinks.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import cx from 'classnames';
+
+import Link from 'Link';
+
+type QuickLinksProps = {
+  className?: string;
+  items: {
+    href: string;
+    label: string;
+  }[];
+};
+
+export const QuickLinks = ({ className, items }: QuickLinksProps) => {
+  const classNames = cx('c-quick-links', {
+    className
+  });
+  return (
+    <ul className={classNames}>
+      {items.map(({ href, label }) => (
+        <li className="c-quick-links__item">
```

Comment:
I think we need a `key` prop here.

Reply:
I'll add this in

Reply:
Updated, also I learned that React keys only need to be unique between siblings. I didn't know this before.

https://www.digitalocean.com/community/tutorials/react-react-keys-and-you#keys-must-be-unique-but-only-amongst-siblings